messybrainz/db/recording.py

```diff
@@ -0,0 +1,149 @@
+from messybrainz import db
+from sqlalchemy import text
+
+
+def insert_recording_cluster(connection, cluster_id, recording_gids):
+    """Creates new cluster in the recording_cluster table.
+
+    Args:
+        connection: the sqlalchemy db connection to be used to execute queries
+        cluster_id (UUID): the recording MSID which will represent the cluster.
+        recording_gids (list): the list of MSIDs will form a cluster.
+    """
+
+    query = text("""INSERT INTO recording_cluster (cluster_id, recording_gid, updated)
+                         VALUES (:cluster_id, :recording_gid, now())
+    """)
+
+    values = [
+        {"cluster_id": cluster_id, "recording_gid": recording_gid} for recording_gid in recording_gids
+    ]
+    connection.execute(query, values)
+
+
+def fetch_gids_for_recording_mbid(connection, recording_mbid):
+    """Fetches the gids corresponding to a recording_mbid that are
+       not present in recording_cluster table.
+
+    Args:
+        connection: the sqlalchemy db connection to be used to execute queries
+        recording_mbid (UUID): the recording MBID for which gids are to be fetched.
+
+    Returns:
+        List of gids.
+    """
+
+    query = text("""SELECT r.gid
```

Comment:
just a small nitpick as I noticed this in the last PR as well, instead of saving the query in a variable and then calling `connection.execute(query, args)`, using  ```connection.execute(text("""SELECT * FROM recording"""), args)``` would be much more readable.